assets/src/application/ui/layout/descriptionList/styles.scss

```diff
@@ -1,71 +1,70 @@
-.ee {
-	&-description-list {
-		&-grid {
-			display: grid;
-			grid-template-columns: max-content;
-			grid-template-rows: repeat(2, var(--ee-font-size-huge));
-			grid-auto-flow: column;
-			justify-content: start;
-			line-height: var(--ee-font-size-default);
-			place-items: center start;
-			margin-top: var(--ee-margin-small);
-			width: 100%;
+.ee-description-list {
```

Comment:
changed this after doing a search for `ee-description-list` and getting nothing :(

Reply:
the:
```
.ee {
	&-description-list {
		&-grid {
```
was resulting in [`ee-description-list-grid`](https://github.com/eventespresso/event-espresso-core/blob/EDTR/master/assets/src/application/ui/layout/descriptionList/DescriptionList.tsx#L9), no styles has been applied to `ee-description-list`;
the change you've made is fine, since it's avoiding one nesting level :+1: 
***
I think I started `.ee {` namespacing when there was multiple blocks in the same stylesheet.